### PR TITLE
Add kedro

### PR DESCRIPTION
**Pull request by marcelotrevisani**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

cc: @lorenabalan

Related to #11316 and #10938

### CODE REVIEW

File: recipes/kedro/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    Kedro is a development workflow framework that implements
+    software engineering best-practice for data pipelines
+    with an eye towards productionising machine learning models.
+
+extra:
+  recipe-maintainers:
+    - marcelotrevisani
+    - lorenabalan
+    - yetudada
+    - idanov
```
Comment (marcelotrevisani) [inline recipes/kedro/meta.yaml:66]: @lorenabalan , @yetudada , @idanov
Could you please confirm that you would like to maintain this recipe as wel?

Comment (yetudada) [reply recipes/kedro/meta.yaml:66]: I can confirm! Thank you @marcelotrevisani!

**Comment by marcelotrevisani** [comment]
@conda-forge/staged-recipes please review :)

File: recipes/kedro/meta.yaml (hunk 1 above)
Comment (lorenabalan) [reply recipes/kedro/meta.yaml:70]: I confirm! Thanks for all your help @marcelotrevisani !

Comment (marcelotrevisani) [reply recipes/kedro/meta.yaml:70]: Just waiting for the confirmation of @idanov

**Comment by marcelotrevisani** [comment]
Thanks!

File: recipes/kedro/meta.yaml (hunk 1 above)
Comment (idanov) [reply recipes/kedro/meta.yaml:70]: I confirm. Thanks a lot @marcelotrevisani!

**Pull request merged**